packages/browser-test-runner/karma-setup.js

```diff
@@ -36,22 +38,29 @@ jest.runOnlyPendingTimers = timers.runOnlyPendingTimers
 jest.setSystemTime = timers.setSystemTime
 jest.useFakeTimers = timers.useFakeTimers
 jest.useRealTimers = timers.useRealTimers
+
 // eslint-disable-next-line no-undef
 jasmine.DEFAULT_TIMEOUT_INTERVAL = 15000
+
 // eslint-disable-next-line no-underscore-dangle
 jest._checkFakeTimers = timers._checkFakeTimers
 
+// eslint-disable-next-line no-undef
+jasmine.getEnv().configure({ random: false }) // disable random test order
+
 Object.assign(jest, timers)
 
 expect.extend(jestExtendedMatchers)
 
 // Add missing Jest functions
 window.test = window.it
 window.test.each = (inputs) => (testName, test) =>
-    inputs.forEach((args) => window.it(testName, () => test(...args)))
+    inputs.forEach((args) => window.it(format(testName, args), () => test(args)))
```

Comment:
Is the removal of `...` intentional? Does it fix a bug in this test setup script?